src/components/structures/TimelinePanel.js

```diff
@@ -1178,11 +1168,11 @@ const TimelinePanel = React.createClass({
                 adjacentInvisibleEventCount = 0;
             }
 
-            const shouldIgnore = (ignoreEchoes && ev.status) || // local echo
+            const shouldIgnore = !!ev.status || // local echo
```

Comment:
Hmm, so I guess this change read **receipt** handling to now also ignore local echoes, but hopefully that's okay...?

Reply:
Oh, good point. I assumed that `this.state.liveEvents` wouldn't include pending events, but looking at `_getEvents`, they do. Will fix!

Reply:
Oh, my comment above doesn't matter, as with this change, we are always ignoring local echoes here as you pointed out. We indeed should never send a read receipt for an event that hasn't been sent yet, so yes, this should be ok.

Reply:
Right, thanks for looking!